public/src/components/linkTracking/MediumSelector.tsx

```diff
@@ -0,0 +1,249 @@
+      { value: 'sother', label: 'Other (osother)' },
+    ],
+  },
+  {
+    group: 'OUTDOOR',
+    options: [
+      { value: 'dooh', label: 'Digital OOH (dooh)' },
+      { value: 'pos', label: 'Point of Sale (pos)' },
+      { value: 'tcp', label: 'Tube Card Panels (tcp)' },
+      { value: 'tvis', label: 'Transvision (tvis)' },
+    ],
+  },
+  {
+    group: 'PAID_SOCIAL',
+    options: [
+      { value: 'sfbk', label: 'Social Facebook (sfbk)' },
+      { value: 'sisgm', label: 'Social Instagram (sisgm)' },
+      { value: 'slkn', label: 'Social Linkedin (slkn)' },
+      { value: 'spint', label: 'Social Pinterest (spint)' },
+      { value: 'quora', label: 'Social Quora (quora)' },
+      { value: 'reddit', label: 'Social Reddit (reddit)' },
+      { value: 'stik', label: 'Social TikTok (stik)' },
+      { value: 'stwr', label: 'Social Twitter (stwr)' },
+      { value: 'sytb', label: 'Social YouTube (sytb)' },
+      { value: 'sother', label: 'Other (osother)' },
+    ],
+  },
+  {
+    group: 'PPC',
+    options: [
+      { value: 'pdsbg', label: 'Paid search Bing (pdsbg)' },
+      { value: 'pdsge', label: 'Paid search Google (pdsge)' },
+      { value: 'pmax', label: 'Performance Max (pmax)' },
+    ],
+  },
+  {
+    group: 'PRINT',
+    options: [
+      { value: 'dirm', label: 'Direct Mail (dirm)' },
+      { value: 'inpaper-g', label: 'Inpaper (inpaper-g)' },
+      { value: 'insr', label: 'Inserts (insr)' },
+      { value: 'prsctrlm', label: 'Press Control Message (prsctrlm)' },
+      { value: 'prssupm', label: 'Press Supporting Message (prssupm)' },
+      { value: 'smpbr', label: 'Sampling Brochure (smpbr)' },
+    ],
+  },
+  {
+    group: 'RETAIL',
+    options: [{ value: 'urtr', label: 'Unknown retail (urtr)' }],
+  },
+  {
+    group: 'TELEMARKETING',
+    options: [{ value: 'ctcter', label: 'Contact Center (ctcter)' }],
+  },
+  {
+    group: 'UNKNOWN',
+    options: [{ value: 'other', label: 'other (other)' }],
+  },
+  {
+    group: 'VIDEO',
+    options: [
+      { value: 'a4', label: 'All4 (a4)' },
+      { value: 'itvh', label: 'ITVHub (itvh)' },
+      { value: 'preroll', label: 'Preroll (preroll)' },
+      { value: 'sky', label: 'Sky (sky)' },
+    ],
+  },
+];
+
+const useStyles = makeStyles(({ spacing }: Theme) => ({
+  groupHeading: {
+    fontWeight: 700,
+  },
+  item: {
+    marginLeft: spacing(2),
+  },
+}));
+
+interface Props {
+  control: Control;
+}
+
+export const MediumSelector: React.FC<Props> = ({ control }: Props) => {
+  const classes = useStyles();
+
+  return (
+    <FormControl>
+      <Controller
+        name="medium"
+        rules={{ required: true }}
+        as={
+          <Select error={!!control.formState.errors?.medium}>
+            {OPTIONS.map(group => {
+              const groupItem = (
+                <MenuItem
+                  className={classes.groupHeading}
+                  value={group.group}
+                  key={group.group}
+                  disabled
```

Comment:
MUI's `Select` doesn't have a way to group items, but we can hack it by using a disabled `MenuItem` with different styling -
![Screenshot 2024-02-28 at 16 01 26](https://github.com/guardian/support-admin-console/assets/1513454/7af12ff5-b55a-467d-b1bd-80698368ec87)